fluid/CodeEditor.h

```diff
@@ -58,6 +55,7 @@ class CodeEditor : public Fl_Text_Editor {
   // attempt to make the fluid code editor widget honour textsize setting
   void textsize(Fl_Fontsize s);
 
+  friend class StyleParse;
```

Comment:
I would probably have put the 'friend' statement at the top of the class for better "visibility", but that's a minor issue.

Update copyright year?

Reply:
Done in eea585760. Are we supposed to manage that whenever editing code, or is it something we do en-masse during releases?